md_api/src/main/java/com/wipro/ats/bdre/md/api/RegisterFile.java

```diff
@@ -52,8 +56,6 @@ public RegisterFile() {
      * batch-id with their respective notation on command line.
      * @return This method returns same input data as instance of RegisterFIleInfo class.
      */
-    @Autowired
-    private RegisterFileDAO registerFileDAO;
 
     public RegisterFileInfo execute(String[] params) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1161)